### PR TITLE
utopiops 0.0.3 (new formula)

### PR DESCRIPTION
**Pull request by Armingodiz**
- [X ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ X] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/utopiops.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+class Utopiops < Formula
+  desc "CLI tool for utopiops.com"
+  homepage "https://www.utopiops.com/"
+  url "https://github.com/utopiops/homebrew-utopiops/releases/download/fresh/utopiops-0.0.3.tar.gz"
```
Comment (carlocab) [inline Formula/utopiops.rb:4]: This is a pre-built binary, which isn't allowed in Homebrew/core. We should build this from source instead. See https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae.

File: Formula/utopiops.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+  sha256 "76267f9feff31860ba0d576a09075c1afb7e0e59266ab89037bf4db79301b364"
+
+  def install
+    bin.install "utopiops"
+  end
+
+  test do
+    system "#{bin}/utopiops", "--version"
```
Comment (carlocab) [inline Formula/utopiops.rb:12]: Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

File: Formula/utopiops.rb (hunk 1 above)
Comment (Armingodiz) [reply Formula/utopiops.rb:4]: Thanks, but my source code is not open source so what should I do?

Comment (carlocab) [reply Formula/utopiops.rb:4]: Sorry, but we only package open source software in Homebrew/core: https://docs.brew.sh/Formula-Cookbook#fill-in-the-license

Comment (Armingodiz) [reply Formula/utopiops.rb:4]: Thanks for your time.

**Pull request closed without merging**